h2o-docs/src/product/security.rst

```diff
@@ -493,7 +493,7 @@ Example **ldap.conf**:
 ::
 
     ldaploginmodule {
-        org.eclipse.jetty.plus.jaas.spi.LdapLoginModule required
+        ai.h2o.org.eclipse.jetty.jaas.spi.LdapLoginModule required
```

Comment:
@michalkurka I'm not sure about this one. Should we mention here 
`ai.h2o.org.eclipse.jetty.jaas.spi.LdapLoginModule` or `org.eclipse.jetty.jaas.spi.LdapLoginModule`

Reply:
@michalkurka I updated this `org.eclipse.jetty.jaas.spi.LdapLoginModule` as I think that is the correct value here